examples/github-appcards/README.md

```diff
@@ -54,19 +58,49 @@ The structure below outlines the most important files in the project used for it
 
 ## Required Environment Variables
 
+To run the app, Vite loads an `.env` file with the [environment variables](https://vitejs.dev/guide/env-and-mode.html#env-files).
+Use the commented example below to create your `.env` file.
+
 ```.env
+# Generate an access token in GitHub, and enter the value here.
+# To generate the access token, go to https://github.com/settings/tokens
```

Comment:
```suggestion
# [Generate an access token in GitHub](https://github.com/settings/tokens), and enter the value here.
```

Reply:
Markdown markup does not work inside code blocks, it renders as literal